demosplan/DemosPlanCoreBundle/Entity/Document/Elements.php

```diff
@@ -103,8 +105,11 @@ class Elements extends CoreEntity implements UuidEntityInterface
      * @var string|null
      *
      * @ORM\Column(name="_e_id", type="string", length=36, options={"fixed":true})
+     *
      * @ORM\Id
+     *
      * @ORM\GeneratedValue(strategy="CUSTOM")
+     *
```

Comment:
@graupnerdemos And why is the CS Fixer in turn ripping these definitions apart?!